storybook/stories/TransactionsList/index.js

```diff
@@ -24,17 +24,43 @@ StoryTransactionsList.defaultProps = {
   transactions: [
     {
       address: 'test address',
-      amount: 1,
+      amount: 0.001,
+      confirmations: 0,
+      symbol: 'test symbol',
```

Comment:
Symbol in this case is one of ETH, BTC, BCC, WAVES, LHT, LTC, NEM.

Symbol is an acronym of a blockchain. Ethereum = ETH, Bitcoin = BTC, Bitcoin Cash = BCC, Litecoin = LTC, Waves = WAVES, Labor Hours Token = LHT, Nem = NEM etc.